study-builder/studies/osteo/subs.conf

```diff
@@ -0,0 +1,11 @@
+{
+  "contact": {
+    "email": "info@osproject.org",
+    "phone": "651-602-2020"
+  },
+  "social": {
+    "twitter": "https://twitter.com/the_osproject/",
+    "facebook": "https://www.facebook.com/osteosarcomaproject/",
```

Comment:
If Ropes ever changes osteo's facebook account, we just need to change this one line and run `--update-emails`. That's it!

Reply:
CC @fat23cat

Reply:
Woot!